doc/content.adoc

```diff
@@ -311,26 +311,38 @@ user defined types to jooq/jdbc compatible types, and other for backwards conver
 .Let see an example for adapt clojure persistent map interface to postgresql json file.
 [source, clojure]
 ----
-(require '[suricatta.proto :as proto])
-(require '[cheshire.core :as json])
+(require '[suricatta.proto :as proto]
+         '[suricatta.impl :as impl]
+         '[cheshire.core :as json])
+
 (import 'org.postgresql.util.PGobject)
 
 (extend-protocol proto/IParamType
   clojure.lang.IPersistentMap
-  (-render [self]
-    (str "'" (json/generate-string self) "'::json"))
-  (-bind [self stmt index]
-    (let [obj (doto (PGobject.)
-                (.setType "json")
-                (.setValue (json/generate-string self)))]
-      (.setObject stmt index obj))))
+  (-render [self ctx]
+    (if (impl/inline? ctx)
+      (str "'" (json/encode self) "'::json")
+      "?::json"))
+
+  (-bind [self ctx]
+    (when-not (impl/inline? ctx)
+      (let [stmt (.statement ^BindContext ctx)]
```

Comment:
This code wont compile because of extra bracket.
Also, `BindContext` is a `jooq` class, I don't think it's a good idea to leak third-party dependencies into client code, at least I wouldn't do that.